main.go

```diff
@@ -119,6 +123,12 @@ func processExcludePods() (map[string]string, error) {
 }
 
 func processTaint() (*v1.Taint, error) {
+	if len(*annotationVar) != 0 {
+		return nil, fmt.Errorf("Annotation must not be specified when taints are specified")
```

Comment:
Good catch. Fixed the bug. 

> Also, since taintVar has a default value, is the expectation that you manually set it to "" by providing the flag? Seems.. not great.

Fair enough. Making all defaults empty.

> It's also undiscoverable what annotation I am supposed to provide. Would any string work? Is there a special string?

It is on purpose. This module is configurable as it is. In GKE we may choose to use specific annotations that suit our needs over time.